gauntlet/packages/gauntlet-solana/src/commands/internal/solana.ts

```diff
@@ -86,9 +161,15 @@ export default abstract class SolanaCommand extends WriteCommand<TransactionResp
       price?: number
     } = {},
   ): Promise<TransactionSignature> => {
-    const { blockhash, lastValidBlockHeight } = await this.provider.connection.getLatestBlockhash()
     if (overrides.units) logger.info(`Sending transaction with custom unit limit: ${overrides.units}`)
     if (overrides.price) logger.info(`Sending transaction with custom unit price: ${overrides.price}`)
+
+    // let confirmLevel: TransactionConfirmationStatus = 'confirmed'
+    let confirmLevel: TransactionConfirmationStatus = 'finalized'
```

Comment:
I'm still debugging this, but I think `simulateTransaction` available in web3.js defaults to using `finalized`, so if sending a sequence of transactions with `confirmed`, the following simulate might run too soon and not see previous changes. 

I have both a workaround as well as work towards an update: `simulateTransaction` has a new version with a confirmation level, but it requires the new transaction version (instead of legacy). I'll try to get those changes in